apps/system/style/chrome/chrome.css

```diff
@@ -308,6 +308,19 @@
   background-image: url("images/light/ssl_broken.png");
 }
 
+.site-icon {
+  width: 3.2rem;
+  height: 3.2rem;
+  background: no-repeat center / 3.2rem content-box;
+  background-image: url('images/default_icon.png');
+  position: absolute;
```

Comment:
can we border-radius all the time while we iron out the final visuals?
the square icons are stressing me out :)